pkgs/development/python-modules/pytest-rerunfailures/default.nix

```diff
@@ -1,23 +1,22 @@
-{ lib, buildPythonPackage, pythonOlder, fetchPypi, pytest, mock }:
+{ lib, buildPythonPackage, pythonOlder, fetchPypi, setuptools, pytest, pytestCheckHook, mock }:
 
 buildPythonPackage rec {
   pname = "pytest-rerunfailures";
-  version = "10.3";
+  version = "11.0";
+  format = "pyproject";
 
-  disabled = pythonOlder "3.5";
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-2CRNeZ+Jpu215XMB3a6ztvENZpFjjVHoCzcTEVkuKMY=";
+    hash = "sha256-zuWeTm8Nc65j+6CZhlMUupuRW3GTSVQREPoBL/tu+xM=";
   };
 
-  buildInputs = [ pytest ];
+  nativeBuildInputs = [ setuptools ];
 
-  nativeCheckInputs = [ mock pytest ];
+  buildInputs = [ pytest ];
 
-  checkPhase = ''
-    py.test test_pytest_rerunfailures.py
-  '';
+  nativeCheckInputs = [ mock pytestCheckHook ];
```

Comment:
The dependency on mock was dropped in 9.1.

https://github.com/pytest-dev/pytest-rerunfailures/blob/ae55e1a5e832f3e5f5e84a28b95d0e908d33af9b/CHANGES.rst#other-changes-1

Reply:
Indeed. [Removed](https://github.com/NixOS/nixpkgs/compare/0d1bad0fbc58697bfac4babaf5816010cba72a57..1c9241f1f5a72ef1b464ebf621a545bb388b8e49) now.